foundry/pixi/containers/wallsLayer.d.ts

```diff
@@ -0,0 +1,155 @@
+import 'pixi.js';
+
+declare type WallData = { c: number[], move: number, sense: number, door: number };
+
+/**
+ * The Walls canvas layer which provides a container for Wall objects within the rendered Scene.
+ * @extends {PlaceablesLayer}
```

Comment:
Most of the doc comments in this file will likely give you eslint errors. Please check and fix them. When you document function parameters, please align the dashes.